presto-pinot-toolkit/src/test/java/com/facebook/presto/pinot/query/TestPinotPlanOptimizerSql.java

```diff
@@ -30,4 +39,28 @@ public void assertUsingSqlSyntax()
     {
         Assert.assertEquals(defaultSessionHolder.getConnectorSession().getProperty("use_pinot_sql_for_broker_queries", Boolean.class).booleanValue(), true);
     }
+
+    @Test
+    public void testDistinctLimitPushdown()
+    {
+        PlanBuilder planBuilder = createPlanBuilder(defaultSessionHolder);
+        PlanNode originalPlan = distinctLimit(
+                planBuilder,
+                ImmutableList.of(new VariableReferenceExpression("regionid", BIGINT)),
+                50L,
+                tableScan(planBuilder, pinotTable, regionId));
+        PlanNode optimized = getOptimizedPlan(planBuilder, originalPlan);
+        assertPlanMatch(optimized, PinotTableScanMatcher.match(pinotTable, Optional.of("SELECT regionId FROM hybrid GROUP BY regionId LIMIT 50"), Optional.of(false), originalPlan.getOutputVariables(), useSqlSyntax()), typeProvider);
```

Comment:
nit: split this into multiple lines if possible please.

Reply:
done